### PR TITLE
Add instructions for running the Pony Sync meeting

### PR DESCRIPTION
**Pull request by aturley**
The instructions are intended for someone who is going to run the Pony
Sync meeting. It has a list of the accounts and software that are
necessary for running the meeting, as well a steps for running the
meeting and uploading the recording of the meeting.

### CODE REVIEW

File: pony-sync.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Pony Sync
+
+The Pony Sync is a weekly meeting where Pony contributors discuss topics related to Pony, including recently added or modified issues.
+
+## Running the Pony Sync
+
+The Pony Sync takes place as a [Zoom](https://zoom.us) call. In order to run the Pony Sync you must have Zoom installed, and be logged in as `ponylang.main@gmail.com`. A list of recently modified issues can be generated using the [pony-sync-helper]() application; this list should be pasted into the `sync` stream in Zulip, in a topic whose name is the date of the meeting in US date format (for example May 12, 2020 would be "5/12/2020").
```
Comment (SeanTAllen) [inline pony-sync.md]: you are missing the link to the sync helper app here.

File: pony-sync.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+### Prerequisites
+
+In order to run the Pony Sync meeting you will need:
+
+1. A GitHub account
+1. A GitHub personal access token for running the `pony-sync-helper` program
+1. The [`pony-sync-helper`](https://github.com/aturley/pony-sync-helper) program
```
Comment (SeanTAllen) [inline pony-sync.md]: Not directly related:

I think it makes sense to transition the sync helper to a ponylang org controlled project. Would you be open to that Andy?

File: pony-sync.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Running the Pony Sync
+
+The Pony Sync takes place as a [Zoom](https://zoom.us) call. In order to run the Pony Sync you must have Zoom installed, and be logged in as `ponylang.main@gmail.com`. A list of recently modified issues can be generated using the [pony-sync-helper]() application; this list should be pasted into the `sync` stream in Zulip, in a topic whose name is the date of the meeting in US date format (for example May 12, 2020 would be "5/12/2020").
+
+### Prerequisites
+
+In order to run the Pony Sync meeting you will need:
+
+1. A GitHub account
```
Comment (SeanTAllen) [inline pony-sync.md:13]: I'm not a fan of markdown numbered list like this, it makes looking at the source weird.

File: pony-sync.md (hunk 2 above)
Comment (aturley) [reply pony-sync.md]: I agree. That would be fine. I'll do that and then update the links.

File: pony-sync.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+1. Using the `ponylang.main@gmail.com` account, log into the Zoom meeting a few minutes before the official start time.
+1. When you feel that there are enough people in the meeting to start, announce that you will be recording the meeting and give participants a few seconds to stop talking. Give a countdown to when you will start recording so that noone is caught unaware.
+1. Start recording the meeting in Zoom.
+1. Begin the meeting by stating that this is the Pony Sync meeting, giving the date, and giving everyone in the room a chance to introduce themselves.
+1. After the introduction the meeting runner can set the agenda and proceed with the meeting.
+1. When the meeting discussion has ended, stop recording and announce that you have stopped recording.
+1. The recording of the meeting will be written to disk when you leave the meeting.
+1. Use SCP to copy the meeting recording to the `r` directory of the Pony account on [nearlyfreespeach.org](https://nearlyfreespeach.org). The file should be called `DATE.m4a`, where `DATE` is the date in YYYY\_MM\_DD format(for example May 12, 2020 would be `2020_05_12.m4a`).
```
Comment (SeanTAllen) [inline pony-sync.md]: I think we should add something to this document for how to get an account so you can do the nearlyfreespeech upload.

I think that involves me, do you have access to add others Andy?

File: pony-sync.md (hunk 3 above)
Comment (aturley) [reply pony-sync.md:13]: I like it because I don't have renumber the list if I add anything, but I'm not strongly committed to that so I can change it.

Comment (SeanTAllen) [reply pony-sync.md:13]: let's see what others think. I agree with the renumbering. It's why I always use `-` instead.

File: pony-sync.md (hunk 4 above)
Comment (aturley) [reply pony-sync.md]: It doesn't look like I can add others.

Comment (SeanTAllen) [reply pony-sync.md]: I guess that means it has to go through me. A note to that effect would be good.

File: pony-sync.md (hunk 1 above)
Comment (aturley) [reply pony-sync.md]: Fixed.

File: pony-sync.md (hunk 2 above)
Comment (aturley) [reply pony-sync.md]: Done.

File: pony-sync.md (hunk 3 above)
Comment (aturley) [reply pony-sync.md:13]: I made the change.

File: pony-sync.md (hunk 4 above)
Comment (aturley) [reply pony-sync.md]: Done.

File: pony-sync.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+In order to run the Pony Sync meeting you will need:
+
+1. A GitHub account
+2. A GitHub personal access token for running the `pony-sync-helper` program
+3. The [`pony-sync-helper`](https://github.com/ponylang/pony-sync-helper) program
+4. [Zulip](https://zulipchat.com/) and a Zulip account
+5. A 1Password account that is connected to the Pony 1Password account
+6. [Zoom](https://zoom.us)
+7. The Zoom password for the `ponylang.main@gmail.com` account, which is available in the Pony 1Password vault
```
Comment (jemc) [inline pony-sync.md:19]: Should we include a link to the 1Password vault here? I have sometimes forgotten the correct URL when working from a different computer.

File: pony-sync.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+8. Access to the Pony Sync recording account on [nearlyfreespeach.net](https://nearlyfreespeach.net) (for access, contact Sean T. Allen via private message on Zulip)
```
Comment (jemc) [inline pony-sync.md]:
```suggestion
8. Access to the Pony Sync recording account on [nearlyfreespeech.net](https://nearlyfreespeech.net) (for access, contact Sean T. Allen via private message on Zulip)
```

File: pony-sync.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+2. Using the `ponylang.main@gmail.com` account, log into the Zoom meeting a few minutes before the official start time.
+3. When you feel that there are enough people in the meeting to start, announce that you will be recording the meeting and give participants a few seconds to stop talking. Give a countdown to when you will start recording so that noone is caught unaware.
+4. Start recording the meeting in Zoom.
+5. Begin the meeting by stating that this is the Pony Sync meeting, giving the date, and giving everyone in the room a chance to introduce themselves.
+6. After the introduction the meeting runner can set the agenda and proceed with the meeting.
+7. When the meeting discussion has ended, stop recording and announce that you have stopped recording.
+8. The recording of the meeting will be written to disk when you leave the meeting.
+9. Use SCP to copy the meeting recording to the `r` directory of the Pony account on [nearlyfreespeach.net](https://nearlyfreespeach.net). The file should be called `DATE.m4a`, where `DATE` is the date in YYYY\_MM\_DD format(for example May 12, 2020 would be `2020_05_12.m4a`).
```
Comment (jemc) [inline pony-sync.md]:
```suggestion
9. Use SCP to copy the meeting recording to the `r` directory of the Pony account on [nearlyfreespeech.net](https://nearlyfreespeech.net). The file should be called `DATE.m4a`, where `DATE` is the date in YYYY\_MM\_DD format(for example May 12, 2020 would be `2020_05_12.m4a`).
```

File: pony-sync.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+10. Put a note in the Zulip topic for the sync announcing that the sync recording is available and points to the recording URL.
```
Comment (jemc) [inline pony-sync.md]: A link to the Zulip topic might be nice
```suggestion
10. Put a note in [the Zulip topic for the sync](https://ponylang.zulipchat.com/#narrow/stream/190591-sync) announcing that the sync recording is available and points to the recording URL.
```

File: pony-sync.md (hunk 5 above)
Comment (aturley) [reply pony-sync.md:19]: Added a link above to the item that mentions the 1password account.

**Comment by aturley** [comment]
@SeanTAllen @jemc is there anything else that needs to be done here?

**Comment by SeanTAllen** [comment]
@aturley I'd say update this PR to include a link to this document in the README.md as we have been doing that and then its good to squash and merge.

**Pull request merged**